spaces/domains.json

```
@@ -469,6 +469,7 @@
   "vote.matcha.fi": "streamingfast.eth",
   "vote.bsc-mvb.io": "mvb.eth",
   "vote.dios.finance": "diosfinance.eth",
+  "vote.sunrise.art": "sunrisefund.eth"
```

You will miss a comma at the end of the line.
```suggestion
  "vote.sunrise.art": "sunrisefund.eth",
```